### ISSUE TITLE
Move AriaHerald to scenery-phet

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/scenery-phet#290]
[AriaHerald](https://github.com/phetsims/balloons-and-static-electricity/blob/master/js/balloons-and-static-electricity/accessibility/AriaHerald.js) is being used in balloons-and-static-electricity as a static object that sends alerts and status updates to screen readers.  It should be generalized and moved to common code.  In https://github.com/phetsims/john-travoltage/issues/171 we decided that scenery-phet would be a good place for this static object.

**Comment by pixelzoom** [comment]
@jessegreenberg I see that `AriaHerald` is now in scenery-phet.  Is there anything else to do here?

**Comment by jessegreenberg** [comment]
Thanks, yes, this can be closed.
